### PR TITLE
Added ORC (Common Order Segment) segment, with basic test

### PR DESCRIPTION
**Pull request by ESouza**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -2,9 +2,10 @@
 require 'rubygems'
 require 'rake'
 require 'rake/testtask'
-require 'rake/rdoctask'
+require 'rdoc/task'
 require 'rake/packagetask'
-require 'rake/gempackagetask'
+#require 'rake/gempackagetask'
```
Comment (segfault) [inline Rakefile]: looks good and passes tests for me. can you remove the commented code and then I'll merge.

**Comment by ESouza** [comment]
Which commented code?

**Comment by ESouza** [comment]
Ok got it. Just did.

**Comment by ESouza** [comment]
Perfect thanks. Could you please publish on RubyGems when you have a chance?

**Pull request merged**